### PR TITLE
Standalone CI docker container

### PR DESCRIPTION
**Pull request by crohr**
Follow-up on #8858, which stops using selenium hub since it increases the number of flaky tests. Now using a single standalone docker image, configured to be able to run tests in parallel.

### CODE REVIEW

**Review by oliverguenther — APPROVED** [review]
Good to merge from my side with the conflicts resolved :+1:

**Comment by crohr** [comment]
@oliverguenther Travis fails to run all the features test. I suspect because the puma server doesn't bind to the correct IP maybe?  https://travis-ci.com/github/opf/openproject/jobs/478284330

Do you have any idea?

File: spec/support/capybara.rb [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
+  port = ENV.fetch('CAPYBARA_SERVER_PORT', ENV.fetch('TEST_ENV_NUMBER', '1').to_i + 3000).to_i
   if port > 0
     Capybara.server_port = port
   end
   Capybara.always_include_port = true
 
   ip_address = Socket.ip_address_list.find { |ai| ai.ipv4? && !ai.ipv4_loopback? }.ip_address
   hostname = ENV['CAPYBARA_DYNAMIC_HOSTNAME'].present? ? ip_address : ENV.fetch('CAPYBARA_APP_HOSTNAME', 'localhost')
-  Capybara.server_host = '0.0.0.0'
+  Capybara.server_host = ip_address
```
Comment (crohr) [inline spec/support/capybara.rb]: @oliverguenther might be the culprit as to why Travis fails to run the tests?

Comment (oliverguenther) [reply spec/support/capybara.rb]: Might be, you can use `ENV['TRAVIS']` to explicitly set 0.0.0.0 again in that case

**Comment by crohr** [comment]
@oliverguenther any idea why legacy specs would fail due to database foreign key constraints? https://github.com/opf/openproject/runs/1809435314

Is it ok to comment them out for now?

**Pull request merged**